packages/components/src/lib/queries/lexical-query.js

```diff
@@ -218,18 +218,16 @@ export default class LexicalQuery extends Query {
     if (lexiconShortOpts.allow && lexiconShortOpts.allow.length > 0) {
       this.homonym.lexemes.forEach((l) => { l.meaning.clearShortDefs() })
       // Suppress events that will trigger UI messages if source is wordlist
-      if (this._source !== LexicalQuery.sources.WORDLIST) {
-        LexicalQuery.evt.HOMONYM_READY.pub(this.homonym)
-      }
+      LexicalQuery.evt.HOMONYM_READY.pub(this.homonym)
     } else {
       // we won't have any remaining valid short definition requests
       // so go ahead and publish the SHORT_DEFS_READY event so that the UI
       // can update itself
       // but issue the HOMONYM_READY event first otherwise we get errors
       // from the wordlist which expects to see the homonym before definitions
       // Suppress events that will trigger UI messages if source is wordlist
```

Comment:
move the comment about suppressing events to below the homonyn_ready event

Reply:
fixed